app/Http/Controllers/ItemOrderController.php

```diff
@@ -45,7 +59,9 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+        $itemOr = ItemOrder::findOrFail($id);
+
+        return response($itemOr,10);
```

Comment:
The 200 is the status of the transaction and it means success, 10 Has no meaning. Please change it back to 200